examples/README.md

```diff
@@ -68,115 +83,171 @@ The value should look like `HostName=<Host Name>;DeviceId=<Device Name>;SharedAc
 The value should look like `HostName=<Host Name>;SharedAccessKeyName=<Key Name>;SharedAccessKey=<SAS Key>`.
 
-![TwinsSetConst](./example_imgs/TwinsSetConst.png)
+![DirectMethodsSetConst](./example_imgs/DirectMethodsSetConst.png)
 
 9. Click **Build and Force Restart**.
 
-10. Check from the logs in the IDE that the device is registered, connected, and twin's properties are retrieved
+10. Check from the logs in the IDE that the device is registered and connected.
+
+![StartDirectMethodsLogs](./example_imgs/StartDirectMethodsLogs.png)
+
+11. [Call Direct Method](#call-direct-method) (**of the automatically created device**, whose ID you can see in logs) from the Azure Portal and check from the logs in the IDE that the call is received.
+
+![CallDirectMethodsLogs](./example_imgs/CallDirectMethodsLogs.png)
+
+12. In the Azure Portal, check that the result of the call is received.
+
+### Twins Example Setup and Run ###
+
+1. [Login To Azure Portal](#login-to-azure-portal).
+
+2. [Create IoT Hub Resource](#create-iot-hub-resource) (if not created yet).
+
+3. [Create IoT Hub DPS Resource](#create-iot-hub-dps-resource) (if not created yet).
+
+4. [Link The IoT Hub To DPS](#link-an-iot-hub-to-dps).
+
+5. [Create An Individual Enrollment](#create-an-individual-enrollment).
+
+6. [Set up your Imp device](https://developer.electricimp.com/gettingstarted).
+
+7. In the [Electric Imp's IDE](https://ide.electricimp.com) create new Product and Development Device Group.
+
+8. Assign a device to the newly created Device Group.
+
+9. Copy the [Twins example source code](./Twins.agent.nut) and paste it into the IDE as the agent code.
+
+10. Set constants in the agent example code:
+ - *AZURE_DPS_SCOPE_ID*: set the **Scope ID** from the [step 3](#create-iot-hub-dps-resource)
+ - *AZURE_DPS_REGISTRATION_ID*: set the **Registration ID** from the [step 5](#create-an-individual-enrollment)
+ - *AZURE_DPS_DEVICE_KEY*: set the **Device Symmetric Key** from the [step 5](#create-an-individual-enrollment)
+
+![TwinsSetConst](./example_imgs/TwinsSetConst.png)
+
+11. Click **Build and Force Restart**.
+
+12. Check from the logs in the IDE that the device is registered, connected, and twin's properties are retrieved.
 
 ![RetrieveTwinLogs](./example_imgs/RetrieveTwinLogs.png)
 
-11. [Update Twin Document](#retrieveupdate-twin-document) from the Azure Portal: add or change, if already exists, the desired property "test". Then check from the logs in the IDE that the desired properties are received and the reported properties are updated.
+13. [Update Twin Document](#retrieveupdate-twin-document) (**of the device with ID used in the step 5**) from the Azure Portal: add or change, if already exists, the desired property "test". Then check from the logs in the IDE that the desired properties are received and the reported properties are updated.
```

Comment:
@betzrhodes Thanks for the findings! I agree with them.
Screenshots in "*** How To" sections are common and not tied to any concrete example. Because of this, they contain some abstract values instead of the values from examples. I suppose it would be strange to use in "*** How To" sections values from a concrete example. Also if we add a new example, it may contain other values and names.